drivers/vdpa/virtio/virtio_vdpa.c

```diff
@@ -2034,6 +2018,14 @@ virtio_pci_devargs_parse(struct rte_devargs *devargs, int *vdpa, rte_uuid_t vm_u
 			DRV_LOG(ERR, "Failed to parse %s", VIRTIO_ARG_VDPA_STAGE);
 	}
 
+	if (rte_kvargs_count(kvlist, VIRTIO_ARG_VDPA_SOCK_PATH) == 1) {
+		/* stage mode selected, debug only
+		 */
```

Comment:
above comment is wrong due to copy-paste

Reply:
Done